Add launch config and fix npm warning

bkircher opened the pull request:
This PR does not add any user-visible features. It just adds

- a `launch.json` to easily test the extension during development
- add `"licence"` to `package.json` to silence a complaint from npm

bkircher committed:
Add launch configuration

Add a proper launch config that allows testing the extension quickly.

bkircher committed:
Add license field to package.json

Silences a npm warning.

qinezh commented on .vscodeignore:
```diff
@@ -1,4 +1,3 @@
-.vscode/**
```
The files under `.vscode` are not needed at runtime, we should ignore them, see https://code.visualstudio.com/api/working-with-extensions/publishing-extension#.vscodeignore

qinezh commented on .gitignore:
```diff
@@ -1,4 +1,4 @@
-.vscode-test/
\ No newline at end of file
+.vscode-test/
+package-lock.json
```
package-lock.json is intended to be checked into source control as far as I know

qinezh commented:
@bkircher Thanks for your contribution, I added some comments, please take a look : )

bkircher committed:
Amend PR

- do not ignore package-lock.json file, supposed to be checked-in
- ignore .vscode/ directory in editor

bkircher commented:
Both comments are absolutely correct. I added another commit to this PR that addresses both. You can squash them if you decide to merge this PR.

qinezh commented:
LGTM, thanks @bkircher
